containers/Vote/NewProposal/ProposalSubForm/index.js

```diff
@@ -0,0 +1,85 @@
+import { memo } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Card, Grid, Typography } from '@material-ui/core'
+import { Controller } from 'react-hook-form'
+import { useWeb3React } from '@web3-react/core'
+
+import CircleHelpIcon from 'components/Icons/CircleHelpIcon'
+import SnowWalletAccount from 'components/SnowWalletAccount'
+import SnowTextField from 'components/UI/TextFields/SnowTextField'
+import SnowSelect from 'components/UI/TextFields/SnowSelect'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    padding: theme.spacing(1.5, 2.5),
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  }
+}))
+
+const PERIOD_TYPES = [
```

Comment:
This should go to 30 days

Reply:
"days" should be omitted from the options and just included in the field description.

Reply:
1 and 2 are not options. 3 is the minimum

Reply:
How about using number input instead of select?